### PR TITLE
Refacto Improve > International controllers

### LINKED ISSUE #37136: Refacto Improve > International controllers with PrestaShopAdminController
- src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController
- src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController
- src/PrestaShopBundle/Controller/Admin/Improve/International/TaxRulesGroupController
- src/PrestaShopBundle/Controller/Admin/Improve/International/StateController
- src/PrestaShopBundle/Controller/Admin/Improve/International/LocalizationController 
- src/PrestaShopBundle/Controller/Admin/Improve/International/TaxController
- src/PrestaShopBundle/Controller/Admin/Improve/International/GeolocationController
- src/PrestaShopBundle/Controller/Admin/Improve/International/LanguageController
- src/PrestaShopBundle/Controller/Admin/Improve/International/TranslationsController
- src/PrestaShopBundle/Controller/Admin/Improve/International/CountryController
- src/PrestaShopBundle/Controller/Admin/Improve/International/ZoneController

### PR DESCRIPTION
**Pull request by boherm**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto Improve > International controllers with PrestaShopAdminController
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | 1. Go to BO<br>2. Test all features in Improve > International
| UI Tests          | https://github.com/boherm/ga.tests.ui.pr/actions/runs/11361044495
| Fixed issue or discussion?     | Fixes #37136 
| Related PRs       | ~
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Comment by ps-jarvis** [comment]
This pull request seems to contain new translation strings. I have summarized them below to ease up review:

- `Admin.International.Feature`
    - [ ] `The %s field is not valid`
    - [ ] `The %s field is invalid.`

(Note: this is an automated message, but answering it will reach a real human)

**Comment by boherm** [comment]
Tests UI 🟢

File: src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php [hunk 1]
```diff
@@ -49,19 +49,22 @@
 use PrestaShop\PrestaShop\Core\Domain\Currency\QueryResult\ExchangeRate as ExchangeRateResult;
 use PrestaShop\PrestaShop\Core\Domain\Currency\QueryResult\ReferenceCurrency;
 use PrestaShop\PrestaShop\Core\Domain\Currency\ValueObject\ExchangeRate;
+use PrestaShop\PrestaShop\Core\Form\FormHandlerInterface as CoreFormHandlerInterface;
```
Comment (jolelievre) [inline src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php]:
```suggestion
use PrestaShop\PrestaShop\Core\Form\FormHandlerInterface as ConfigurationFormHandlerInterface;
```

**Review by jolelievre — COMMENTED** [review]
Thanks @boherm 

Just one comment

**Review by jolelievre — APPROVED** [review]
Thanks @boherm

**Review by tleon — APPROVED** [review]

**Pull request merged**